storage/interface.go

```diff
@@ -30,6 +30,7 @@ var (
 	ErrDuplicateSampleForTimestamp = errors.New("duplicate sample for timestamp")
 	ErrOutOfBounds                 = errors.New("out of bounds")
 	ErrOutOfOrderExemplar          = errors.New("out of order exemplar")
+	ErrExemplarLabelLength         = errors.New("label length for exemplar exceeds maximum")
```

Comment:
How about this? Quicker to know what is the limit. `ExemplarMaxLabelSetLength` could be moved somewhere here to avoid circular imports.

```suggestion
	ErrExemplarLabelLength         = errors.New(fmt.Sprintf("label length for exemplar exceeds %d characters", ExemplarMaxLabelSetLength))
```

Reply:
makes sense, but `errors.Errorf` instead of New and Sprintf both?